docs/contents/community/faq.md

```diff
@@ -63,8 +64,8 @@
     What operating system can I install EKS Distro on?
 
 !!! quote "Answer"
-    We provide the same upstream versions of Kubernetes and dependencies which 
-    operating system vendors have tested and confirmed to work with Kubernetes,
+    We provide the same upstream versions of Kubernetes and dependencies that 
+    operating system vendors have tested and confirmed to work with Kubernetes
     and will work the same way. As a result, EKS Distro works with common operating
     systems already used to create Kubernetes clusters, such as CentOS, 
     Canonical Ubuntu, Red Hat Enterprise Linux, Suse, and more. We are testing 
```

Comment:
Also this is not a valid sentence, but I didn't know what it was trying to say

```
[....] We are testing 
EKS Distro with select vendors to ensure support on Bottlerocket, Amazon Linux 2,
and Canonical has tested EKS Distro for compatibility with Ubuntu.
```

Reply:
Seems like that should be `EKS Distro is tested by select vendors on various operating systems including Bottlerocket, Amazon Linux 2 and Ubuntu.`